tests/test_data_handling_h5_cc.py

```diff
@@ -91,6 +96,54 @@ def test_solar_handler(plot=False):
                 plt.close()
 
 
+def test_solar_handler_w_wind(plot=False):
+    """Test loading irrad data from NSRDB file and calculating clearsky ratio
+    with NaN values for nighttime. Also test the inclusion of wind features"""
+
+    features_s = ['clearsky_ratio', 'U_200m', 'V_200m', 'ghi', 'clearsky_ghi']
+
+    with tempfile.TemporaryDirectory() as td:
+        res_fp = os.path.join(td, 'solar_w_wind.h5')
+        shutil.copy(INPUT_FILE_S, res_fp)
+
+        with Outputs(res_fp, mode='a') as res:
+            res.write_dataset('windspeed_200m',
+                              np.random.uniform(0, 20, res.shape),
+                              np.float32)
+            res.write_dataset('winddirection_200m',
+                              np.random.uniform(0, 359.9, res.shape),
+                              np.float32)
+
+        handler = DataHandlerH5SolarCC(res_fp, features_s,
+                                       target=TARGET_S, shape=SHAPE,
+                                       temporal_slice=slice(None, None, 2),
+                                       time_roll=-7,
+                                       val_split=0.1,
+                                       sample_shape=(20, 20, 24),
+                                       max_workers=1)
+
+        assert handler.data.shape[2] % 24 == 0
+        assert handler.val_data.shape[2] % 24 == 0
```

Comment:
I dont think this will pass. I changed handler.val_data is be None when val_split = 0.0, and val_split is forced to be 0.0 for this data handler, right?

Reply:
Gotcha yeah, I think i must have missed this during our concurrent dev. fixed.